src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -62,6 +63,10 @@
         <xsd:attribute name="path" type="xsd:string" />
     </xsd:complexType>
 
+    <xsd:complexType name="links">
+        <xsd:attribute name="enabled" type="xsd:boolean" />
```

Comment:
Does it make sense to set this to `xsd:string` to allow using a DI parameter in the config?